Mathlib/LinearAlgebra/AffineSpace/Independent.lean

```diff
@@ -712,7 +712,8 @@ end DivisionRing
 
 section Ordered
 
-variable {k : Type*} {V : Type*} {P : Type*} [LinearOrderedRing k] [AddCommGroup V]
+variable {k : Type*} {V : Type*} {P : Type*} [Ring k] [LinearOrder k] [IsStrictOrderedRing k]
+  [AddCommGroup V]
```

Comment:
[Note to self: I've read up to here and from now on it gets annoying because the diffs aren't loaded automatically]